File: src/main/java/com/alexanderbelldev/trelloclone/Model/Item.java
package com.alexanderbelldev.trelloclone.Model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Item {


    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    Integer id;
    String username;
    String itemName;
    String itemColor;
    String columnName;
    String itemDetail;
    Integer indexNum;
}

File: src/main/trellofrontend/src/app/model/item.ts
export class Item {

  id?: number;
  username: string;
  itemName: string;
  columnName: string;
  itemDetail: string;
  indexNum: number;
  itemColor: string;
  delete: boolean;


  constructor(username: string, itemName: string, columnName: string, itemDetail: string, indexNum: number, itemColor?: string) {
    this.username = username;
    this.itemName = itemName;
    this.columnName = columnName;
    this.itemDetail = itemDetail;
    this.indexNum = indexNum;
    this.itemColor = itemColor;
  }
}
